src/richie/apps/core/context_processors.py

```diff
@@ -32,13 +50,19 @@ def site_metas(request):
             "domain": site_current.domain,
             "web_url": f"{protocol:s}://{site_current.domain:s}",
         },
+        "AUTHENTICATION": {
+            "ENDPOINT": authentication_endpoint,
+            "PROFILE_URLS": json.dumps(profile_urls),
+        },
         "FRONTEND_CONTEXT": json.dumps(
             {
                 "context": {
                     "csrftoken": get_token(request),
                     "environment": getattr(settings, "ENVIRONMENT", ""),
                     "release": getattr(settings, "RELEASE", ""),
                     "sentry_dsn": getattr(settings, "SENTRY_DSN", ""),
+                    "auth_endpoint": authentication_endpoint,
```

Comment:
dulicate?

Reply:
I forgot to remove `AUTHENTICATION.ENDPOINT`.